docs/theme/included/dotmatrix.rst

```diff
@@ -0,0 +1,493 @@
+DotMatrix Theme
+***************
+
+The included ``themes/dotmatrix/dotmatrix.theme`` file defines an original theme design,
+that will completely changes the way you imagine the prompt.
+
+It is focused on *improving the user experience*,
+by vastly improving the ability of the user to immediatly spot
+what she should do following her system's state changes.
+
+Has such, you may find it a little bit disconcerting at first.
+But we believe that the advantages of this design makes it worth the effort.
+
+.. contents::
+   :local:
+
+
+Overview
+========
+
+If there is nothing special about the current context, the appearance of
+DotMatrix might be as simple as this:
+
+.. image:: dotmatrix-short-chevron.png
+   :alt: X   ~
+
+
+A typical display of a slightly complex session would look like this:
+
+.. image:: dotmatrix-med-chevron.png
+
+The DotMatrix theme also comes as several variants, changing the shape of the segments separators,
+like the (default) "dotmatrix" pixels:
+
+.. image:: dotmatrix-short-dots.png
+
+or the "slant" variant:
+
+.. image:: dotmatrix-med-slant.png
+
+
+Design
+======
+
+Rationale
+---------
+
+The DotMatrix theme use a couple of design principles:
+
+* It is accessible to colorblind people, hence it uses:
```

Comment:
"Blue-yellow form" is (by far) the rarest condition, hence why it is recommended for accessibility. But of course, the user may override `DOTMATRIX_COLOR_*` variables.

It would probably be nice to have an option to automatically configure themes in that regard. Maybe a good use-case for a interactive theme configuration bot àla Powerlevel.